source/includes/_one-time-sync-individual-custom-check.html.md.erb

```diff
@@ -0,0 +1,491 @@
+# One-time Sync Individual Custom 
+
+This section lists all endpoints that must be implemented for a one-time synchronous Individual Custom check integration.
+
+One-time synchronous individual custom checks are custom checks with the following behavior:
+
+    * The check result is returned directly upon request; the response is not sent until the check is finished (synchronous).
+    
+    * This one-time check operates at a point in time, meaning the check result is returned once and there is no ongoing monitoring.
+
+
+## Get check configuration for one-time sync Individual custom
+
+<%= partial("includes/partials/config.md.erb", :locals => { :name => "sync_individual_custom" }) %>
+
+This endpoint specifies the configuration used by PassFort for performing a
+one-time synchronous custom check using your integration.
+
+### HTTP request
+
+`GET https://my-integration.example.com/config`
+
+### Response fields
+
+Most configuration fields are already discussed in the [configuration
+section](#configuration). The following options are specific to the Individual custom check.
+
+#### Check type
+
+The `check_type` field must be set to `"INDIVIDUAL_CUSTOM"`.
+
+#### Check template
+
+The `check_template` fields for this check are:
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>type</code></td>
+      <td>The value <code>"ONE_TIME_SYNCHRONOUS"</code></td>
+      <td>Yes</td>
+      <td>
+        Which check template to use for this check. For one-time synchronous
+        checks, you must specify <code>"ONE_TIME_SYNCHRONOUS"</code>.
+      </td>
+    </tr>
+    <tr>
+      <td><code>timeout</code></td>
+      <td>number</td>
+      <td>No</td>
+      <td>
+        The minimum number of seconds Passfort waits without a response before
+        considering identity checks run through this integration to have
+        timed out. Must be an integer. If not specified or set to
+        <code>null</code>, a timeout of 60 seconds is used.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<%= partial("includes/partials/authenticated.md.erb", :locals => {
+  :role => :server,
+}) %>
+
+## Run a one-time sync Individual custom check
+
+This endpoint exposes your data provider's check results to Passfort.
+
+### HTTP request
+
+`POST https://my-integration.example.com/checks`
+
+>A JSON payload following this structure is sent to the endpoint:
+
+```json
+<%= pretty_json("requests/run_check_sync_individual_custom", "json") %>
+```
+
+The payload of the request can contain the following fields:
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Always present?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>id</code></td>
+      <td>UUID</td>
```

Comment:
Might be a stupid question, but why do we tag "id" as code and we leave "UUID" without the tag?